### ISSUE TITLE
Error page requires scrolling on mobile because of extra whitespace

### ISSUE THREAD

**Issue by daprice** [mastodon/mastodon#25676]
### Steps to reproduce the problem

1. Try to open the web interface of a mastodon server that is down

### Expected behaviour

Error page shouldn’t have unnecessary spacing that pushes the text off the bottom of the screen

### Actual behaviour

Error page has unnecessary spacing that pushes the text off the bottom of the visible screen area

### Detailed description

It looks like something on the page is sized using `vh` or `lvh` (the maximum, full screen viewport height) instead of `svh` (the smaller viewport height when the browser chrome is visible). Since users are unlikely to view the error page expanded to full screen, it scrolls unnecessarily and, on some devices, puts the text below the fold.

On my iPhone (iOS 16, Safari) there’s plenty of space to fit the entire image and text on the screen, but the extra spacing makes it tall enough that the text is partially off screen and requires scrolling to see:
![IMG_5717](https://github.com/mastodon/mastodon/assets/3615519/4026052b-68ac-417d-a6d3-2e03271f2f72)

Even on an iPad, the text fits on screen but the page still has enough extra height that it scrolls unnecessarily.

### Specifications

Mastodon v4.1.2+nightly-20230627

Safari on iOS and iPadOS 16.5.1; likely occurs in other browsers and OSs as well

**Comment by forsamori** [comment]
I agree, we should really be displaying this more succinctly - much better UX. I'll pick this one up :)

**Comment by forsamori** [comment]
https://github.com/mastodon/mastodon/pull/25708 PR up for this now :)
